packages/components/psammead-media-indicator/src/index.jsx

```diff
@@ -42,54 +49,44 @@ const TimeDuration = styled.time`
   margin: 0 ${GEL_SPACING_HLF};
 `;
 
-const IndexAlsosMediaIndicator = styled.span`
-  & > svg {
-    margin: 0;
-  }
-`;
-
 const MediaIndicator = ({
   datetime,
   duration,
   type,
   topStory,
   service,
-  indexAlsos,
-}) =>
-  indexAlsos ? (
-    <IndexAlsosMediaIndicator aria-hidden="true">
+  isInline,
+}) => (
+  <MediaIndicatorWrapper
+    aria-hidden="true"
+    topStory={topStory}
+    service={service}
+    isInline={isInline}
+  >
+    <FlexWrapper>
       {mediaIcons[type]}
-    </IndexAlsosMediaIndicator>
-  ) : (
-    <MediaIndicatorWrapper
-      aria-hidden="true"
-      topStory={topStory}
-      service={service}
-    >
-      <FlexWrapper>
-        {mediaIcons[type]}
-        {duration && datetime && (
-          <TimeDuration dateTime={datetime}>{duration}</TimeDuration>
-        )}
-      </FlexWrapper>
-    </MediaIndicatorWrapper>
-  );
+      {duration && datetime && (
+        <TimeDuration dateTime={datetime}>{duration}</TimeDuration>
+      )}
+    </FlexWrapper>
```

Comment:
Can we pass in `children` into the flex wrapper just after the duration? This will enable us to have text that is aligned with the Media Indicator icon, without having to add more divs/spans in our markup.